acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -283,6 +283,257 @@ func TestAPIGateway_Basic(t *testing.T) {
 	}
 }
 
+func TestAPIGateway_JWTAuth_Basic(t *testing.T) {
+	t.Skip()
```

Comment:
Would add a note about why we're skipping for now and will enable once implementation is complete